librz/core/cbin.c

```diff
@@ -368,6 +368,23 @@ RZ_API bool rz_core_bin_apply_all_info(RzCore *r, RzBinFile *binfile) {
 	return true;
 }
 
+RZ_API bool rz_core_file_reload(RzCore *r, const char *file, ut64 baseaddr) {
```

Comment:
Could you please add a Doxygen for this function? Also, I think it's better to call the first parameter as `RzCore *core` for clarity. 
- https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation